backend/register_lambda.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+
+dynamodb = boto3.client('dynamodb')
+
+def lambda_handler(event, context):
+    params = event['queryStringParameters']
```

Comment:
This might get modified to be the body rather than query string params but this is fine for now

Reply:
Sounds good! We can fix in a follow up diff